test-environment.yml

```diff
@@ -64,3 +64,4 @@ dependencies:
   - tabulate
   - retry
   - graphviz
+  - bash >=4.0
```

Comment:
On windows, this seems to be called m2-bash. But I suggest to just not add bash here, since the test case anyway uses its own env. And the bash part of the test case can be moved into a separate test case and restricted to just run on linux (use the skip_on_windows decorator).